app/helpers/frontend/feeds_navigation_bar_structure_helper.rb

```diff
@@ -0,0 +1,63 @@
+module Frontend
+  module FeedsNavigationBarStructureHelper
+    # rendered by _navbar_feeds.haml
+    def feed_structure
+      menu = [
+          { :left =>{ :content => 'News', :href => '/news.atom' } },
```

Comment:
I would really appreciate if we could shrink down the available options. They are not understandable for Joe Average user. For reference: this is the menu built by this code:
![screenshot from 2017-05-20 12-23-53](https://cloud.githubusercontent.com/assets/142237/26275038/23880bf2-3d58-11e7-9534-59064600e3fc.png)

I, despite knowing the code, would not now which feed I actually want. last two xears? last 100? webm or mp4 feed? Furthermore, the Feed named "RSS" is actually an RDF feed - does anyone still use RDF over RSS?

I'd suggest leaving all existing feeds intact but shrinking the menu down to:
- Video Podcast Feed (last 100 HD Videos, MP4)
- Video Archive Feed (all HD Videos, MP4)
- ----
- Audio Podcast Feed (last 100 MP3 Audios)
- Audio Archive Feed (all MP3 Audios)
- ----
- Per-Format-Feeds (all Formats in HD/SD for those who really want to choose for themselfs)